src/content/developers/docs/gas/index.md

```diff
@@ -21,88 +20,104 @@ Since each Ethereum transaction requires computational resources to execute, eac
+Starting with [the London update](/history/#london), every block has a base fee, the minimum per gas price for inclusion in this block. Optionally, users can set a priority fee (tip).
 
-Although a transaction includes a limit, any gas not used in a transaction is returned to the user.
+Calculating the total fee works as follows: `Gas units (limit) * (Base fee + Tip)`
 
-![Diagram showing how unused gas is refunded](../transactions/gas-tx.png)
-_Diagram adapted from [Ethereum EVM illustrated](https://takenobu-hs.github.io/downloads/ethereum_evm_illustrated.pdf)_
+Let’s say Jordan has to pay Taylor 1 ETH. In the transaction the gas limit is 21,000 units and the base fee is 100 gwei. Jordan includes a tip of 10 gwei.
+Using the formula above we can calculate this as `21,000 * (100 + 10) = 2,310,000 gwei` or 0.0023 ETH.
 
-## What is gas limit? {#what-is-gas-limit}
+## Base Fees {#base-fees}
 
-Gas limit refers to the maximum amount of gas you are willing to consume on a transaction. More complicated transactions, involving [smart contracts](/developers/docs/smart-contracts/), require more computational work so they require a higher gas limit than a simple payment. A standard ETH transfer requires a gas limit of 21,000 units of gas.
+Every block has a base fee which acts as a reserve price. To be eligible for inclusion in a block the offered price per gas must at least equal the base fee. The base fee is calculated independently of the current block and is instead determined by the blocks before it - making transaction fees more predictable for users. When the block is mined this base fee is "burned", removing it from circulation.
 
-For example if you put a gas limit of 50,000 for a simple ETH transfer, the EVM would consume 21,000, and you would get back the remaining 29,000. However, if you specify too little gas say for example, a gas limit of 20,000 for a simple ETH transfer, the EVM will consume your 20,000 gas units attempting to fulfill the txn, but it will not complete. The EVM then reverts any changes, but since 20k gas units worth of work has already been done by the miner, that gas is consumed.
+## Block Size {#block-size}
+
+EIP-1559 introduced variable-size blocks to Ethereum. Each block has a target size of 15 million gas but the size of blocks will increase or decrease in accordance with network demands, up until the block limit of 30 milion gas (2x target block size). An equilibrium block size of 15 million on average is achieved through the process of _tâtonnement_. This means if the block size is greater than the target block size, the base fee will increase. Similarly, if the block size is less than the target block size the base fee will decrease. The amount the base fee is adjusted by is proportional to how far from the target the block size is. [More on blocks](/developers/docs/blocks/).
 
-## What is gas price? {#what-is-gas-price}
+## Base Fees Continued {#base-fees-continued}
```

Comment:
Should we just put this section within the "base fee" section above? Seems weird to have "base fee" then "base fees continued". I see how it's a bit tricky because you want to introduce the block size changes as well 🤔  - perhaps we move the section on block size first, then have a single section on base fee?